plugins/modules/foreman_job_template.py

```diff
@@ -249,95 +249,88 @@
 # with name set to "*" bulk actions can be performed
 - name: "Delete *ALL* Job Templates"
   local_action:
-      module: foreman_job_template
-      username: "admin"
-      password: "admin"
-      server_url: "https://foreman.example.com"
-      name: "*"
-      state: absent
+    module: foreman_job_template
+    username: "admin"
+    password: "admin"
+    server_url: "https://foreman.example.com"
+    name: "*"
+    state: absent
 
 - name: "Assign all Job Templates to the same organization(s)"
   local_action:
-      module: foreman_job_template
-      username: "admin"
-      password: "admin"
-      server_url: "https://foreman.example.com"
-      name: "*"
-      state: present
-      organizations:
-      - DALEK INC
-      - sky.net
-      - Doc Brown's garage
+    module: foreman_job_template
+    username: "admin"
+    password: "admin"
+    server_url: "https://foreman.example.com"
+    name: "*"
+    state: present
+    organizations:
+    - DALEK INC
+    - sky.net
+    - Doc Brown's garage
 
 '''
 
 RETURN = ''' # '''
 
-try:
-    from nailgun.entities import (
-        Organization,
-        Location,
-    )
-
-    from ansible.module_utils.ansible_nailgun_cement import (
-        JobTemplate,
-        TemplateInput,
-        find_entities,
-        find_entities_by_name,
-        find_template_input,
-        ForemanEntityAnsibleModule,
-        naildown_entity,
-        naildown_entity_state,
-        sanitize_entity_dict,
-    )
```

Comment:
Where is `os` used? I can't find it in neither the old nor the new code

Reply:
there is some os.path manipulation around line 374.

Reply:
Ooh, I've been fooled by the diff view, didn't see that there were collapsed lines. Or rather not used to see that in a nailgun to apypie PR ;)

Reply:
Maybe now it not the best time anymore to _get used_ to seeing nailgun-to-apypie-prs.